### DISCUSSION TITLE
Gamification

### DISCUSSION THREAD

**Discussion by pdawyndt** [dodona-edu/feedback-and-support#11] [Feature requests]
This is a collection of ideas that could be explored to gamify Dodona:
- badges
  - [bebadges.be](http://www.bebadges.be/) ([article in UGent Magazine "Durf Denken"](http://www.communicatie.ugent.be/export/Durf-Denken/november-2016.pdf))
  - [openbadges.org](http://www.bebadges.be/)
- leaderboards
  - per exercise and/or per track
  - ranking based on criteria that can be set in configuration of exercise or track
- storytelling
  - collaborate with graphical design school (student projects)
  - [Pixar in a Box](https://www.khanacademy.org/partner-content/pixar?utm_source=Sailthru&utm_medium=email&utm_campaign=Pixar%20storytelling%20final%20send&utm_content=C&utm_term=All%20Users)
- problem difficulty
  - stars
  - item response theory

_[Original issue](https://github.ugent.be/dodona/dodona/issues/377) by @pdawyndt on Thu Feb 16 2017 at 08:51._

**Comment by yveshanoulle** [comment]
This is an old issue, so I have no idea how much is yet implemented, a few idea's:

DuoLingo has a lot of gamifications techniques that are attracive. 

- % of people who answered before me ==> this might challenge me to also start 
(maybe with historical data about how early starters have a higher succesrate etc etc)
- streaks (if we want students to lean by doing exercises every day instead of spurts of action, that can really help)
- mistakes reviews is one of my favorite exercises on duolingo. It allows me to practise something again. not sure how that would work here.
- people can cheer for friends on duolingo, it's stupid, but it works. 
we don't see the details, yet we do see levels, streaks, we can compare scores, streaks and all kind of data.

Sometimes I think why do they spend so much time on gamification stuff, but hey it works....
